src/firebolt/async_db/cursor.py

```diff
@@ -327,25 +337,37 @@ async def _api_request(
             content=query,
         )
 
+    async def _async_api_request(self, query: str) -> Response:
+        """Do query request using SET async_execution=1."""
+        return await self._client.request(
+            url="/",
+            method="POST",
+            params={
+                "database": self.connection.database,
+                "output_format": JSON_OUTPUT_FORMAT,
+                **self._set_parameters,
+                "async_execution": 1,
+                "advanced_mode": 1,
+            },
+            content=query,
+        )
+
     async def _do_execute(
         self,
         raw_query: str,
         parameters: Sequence[Sequence[ParameterType]],
-        set_parameters: Optional[Dict] = None,
```

Comment:
This was deprecated already, which is why it was removed. @stepansergeevitch ?

Reply:
Having to still support this AND introduce async queries support would complicate the code a lot.
This has been deprecated for like 5 minor versions, and also users can still stick to the old version until they will fix compatibility on their end.

Reply:
This would mean a breaking change in a minor version though. Should we bump the major instead? Or do we work with the assumption that we might contain breaking changes since we're <1.0?

Reply:
Am I right in that Cosmin ruled against doing this now?